cluster-state-service/internal/features/_test/gucumbertest__.go

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
(multiple occurrences) It looks like you're missing license headers on these files.